tests/v2_validation/cattlevalidationtest/core/test_ldap_auth.py

```diff
@@ -431,6 +477,10 @@ def test_ldap_create_new_env_add_group_member(admin_client):
         project.setmembers(members=new_members)
     assert "object has no attribute" in str(excinfo.value)
 
+    unset_group_search_domain(admin_client)
+    assert len(main_client.list_identity(name=group)) == 0
+    set_group_search_domain(admin_client)
+
```

Comment:
Can you explain what we test here. Seems like we set the config with null groupSearchDomain and check that the group search does not return any group with name 'group'?
And then we set the groupSearchDomain, should there be another search to test that setting the domain makes the group search work?